src/sentry/middleware/superuser.py

```diff
@@ -42,5 +42,17 @@ def process_response(self, request: Request, response: Response) -> Response:
             pass
         su = getattr(request, "superuser", None)
         if su:
+            if su.is_active:
+                org_slug = getattr(getattr(request, "organization", None), "slug", None)
```

Comment:
i'm actually not sure if there's a performance difference, but this could also be written as `request.get('organization', {}).get('slug')` I think 🤷 
could be easier to read